tests/test_settings.py

```diff
@@ -0,0 +1,29 @@
+import os
+from unittest.mock import patch
+
+import pytest
+
+from karmabot.settings import get_env_var
+
+
+@pytest.mark.parametrize("env_var", ["true", "false"])
```

Comment:
from the other pr (I merged `optional_env_variable` branch into this one)

Reply:
👍